espanol/canudos.md

```diff
@@ -51,3 +51,4 @@ el Consejero:
 [Empieza a gritar como un poseso](poseso/poseso.md)
 
 [Se arrodilla y...](rogando/rogando.md)
+[Y vino un ejercito](ejercito/ejercito.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.